### ISSUE TITLE
[Accessibility] Ctrl+M (editor.action.toggleTabFocusMode) does not work inside terminal input field

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#15651]
CC -meganrogge 

-- Reproducible Steps

1. Load screen reader.

1. Open terminal.

1. Focus in terminal input area.

1. Press Ctrl+M to toggle tab-key move (`editor.action.toggleTabFocusMode`).

* Notes: I don't hear any change. It does not seem to work.

-- Expected Behavior

Pressing Ctrl+M key needs to toggle the tab-key move mode.

-- System Info

Version: 1.75.0-insider (system setup)
Commit: f1b09237cd400d92987cf7a9681995e8011ef54c
Date: 2023-01-23T11:53:59.088Z
Electron: 19.1.9
Chromium: 102.0.5005.194
Node.js: 16.14.2
V8: 10.2.154.23-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes
